cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -0,0 +1,211 @@
+    let jsonBody = {
+      description: this.state.profileDescription,
+      provisioner: {
+        name: 'GoogleDataProc',
+        properties: Object.entries(this.properties).map(([property, propObj]) => {
+          return {
+            name: property,
+            value: propObj.value,
+            editable: propObj.editable
+          };
+        })
+      }
+    };
+    console.log(`Coming soon. This is the state right now:
+      ${JSON.stringify(jsonBody, null, 2)}
+    `);
+  };
+
+  renderProfileName = () => {
+    return (
+      <FormGroup row>
+        <Col xs="3">
+          <strong
+            className="label"
+            id="profile-name"
+          >
+            Profile Name
+          </strong>
+        </Col>
+        <Col xs="5">
+          <Input
+            aria-labelledby="profile-name"
+            getRef={ref => this.profileNameInput = ref}
+            value={this.state.profileName}
+            onChange={this.onMetadataChange.bind(this, 'profileName')}
+            placeholder="Add a name for the Compute Profile"
+          />
+        </Col>
+      </FormGroup>
+    );
+  };
+
+  renderDescription = () => {
+    return (
+      <FormGroup row>
+        <Col xs="3">
+          <strong
+            className="label"
+            id="profile-description"
+          >
+            Description
+          </strong>
+        </Col>
+        <Col xs="5">
+          <Input
+            type="textarea"
+            aria-labelledby="profile-description"
+            value={this.state.profileDescription}
+            onChange={this.onMetadataChange.bind(this, 'profileDescription')}
+            placeholder="Add a description for the profile"
+          />
+        </Col>
+      </FormGroup>
+    );
+  };
+
+  renderGroup = (group) => {
+    return (
+      <div className="group-container" key={group.name}>
+        <strong className="group-title"> {group.name} </strong>
+        <hr />
+        <div className="group-description">
+          {group.description}
+        </div>
+        <div className="fields-container">
+          {
+            group.properties.map(property => {
+              let uniqueId = `${group.name}-${property.name}`;
+              return (
+                <FormGroup key={uniqueId} row>
+                  <Col xs="3">
+                    <strong
+                      className="label"
+                      id={uniqueId}
+                    >
+                      {property.label}
+                    </strong>
+                  </Col>
+                  <Col xs="5">
+                    {
+                      <AbstractWidget
+                        type={property['widget-type']}
+                        value={""}
+                        onChange={this.onValueChange.bind(this, property.name)}
+                        widgetProps={property['widget-attributes']}
+                      />
+                    }
+                  </Col>
+                </FormGroup>
```

Comment:
I am not sure if it will be of much use. This function is just iterating over the groups and rendering the widgets. May be when a group becomes a little bit more sophisticated like collapsable groups, some highlights, metadata (#of properties), subtitle etc.,